web/landing/partners.js

```diff
@@ -132,202 +120,119 @@ export function Partners() {
-            mais également limiter la concurrence déloyale très souvent permise
-            par le dumping social.
-          </Typography>
-        </Box>
-        <Box my={10} style={{ textAlign: "justify" }}>
-          <Typography variant="h4" className={classes.title}>
-            Les entreprises partenaires de Mobilic: des entreprises
-            respectueuses des droits sociaux ?
-          </Typography>
-          <Typography>
-            Mobilic{" "}
-            <strong>
-              garantit que son entreprise partenaire répond aux exigences
-              suivantes
-            </strong>{" "}
-            :
-          </Typography>
-          <ul>
-            <li>
-              <Typography>
-                une saisie du temps de travail en temps réel et une traçabilité
-                des modifications qui limitent les possibilités de fraude.
+        <Grid container spacing={15}>
+          <Grid item xs={12} md={6}>
+            <Typography variant="h2" className={classes.title}>
+              Les entreprises
+            </Typography>
+            <Typography className={classes.paragraphDescription}>
+              Les entreprises partenaires{" "}
+              <strong>utilisent Mobilic de manière active</strong>, c'est à dire
+              qu'au moins 75% de leurs salariés inscrits sur Mobilic s'en
+              servent au quotidien.
+            </Typography>
+            <Grid
+              container
+              justifyContent="space-evenly"
+              alignItems="center"
+              spacing={{ xs: 2, md: 4 }}
+            >
+              {partnersSrcs.keys().map(src => (
+                <Grid item key={src}>
+                  <img
+                    alt={src}
+                    src={partnersSrcs(src)}
+                    className={classes.partnerImage}
+                  />
+                </Grid>
+              ))}
+            </Grid>
+            <MainCtaButton
+              aria-label="Devenir partenaire"
+              className={classes.cta}
+              href="mailto:mobilic@beta.gouv.fr"
+            >
+              Devenir partenaire
+            </MainCtaButton>
+          </Grid>
+          <Grid item xs={12} md={6}>
+            <Box marginBottom={16}>
+              <Typography variant="h2" className={classes.title}>
+                Les éditeurs de logiciels
               </Typography>
-            </li>
-            <li>
-              <Typography>
-                une totale transparence auprès des salariés sur leur temps de
-                travail qui leur permet de contrôler la cohérence avec leur
-                fiche de paye.
+              <Typography className={classes.paragraphDescription}>
+                Les éditeurs de logiciels partenaires sont{" "}
+                <strong>
+                  interfacés avec{" "}
+                  <span style={{ whiteSpace: "nowrap" }}>l'API Mobilic :</span>
+                </strong>
+                <ul>
```

Comment:
:green_circle:  Dans la console du navigateur, l'erreur `<ul> cannot appear as a descendant of <p>` apparaît. Il faudrait fermer `Typography` avant.

Reply:
Bonne remarque, par contre, je vais plutôt rajouter un `component="div"` dans le Typography pour tout garder dans le même bloc.